paper.md

```diff
@@ -73,16 +73,16 @@ JOSS: "Begin your paper with a summary of the high-level functionality of your s
 i.e., What does this software do?
 -->
 
-Empirical is a C++ library designed to promote open science and facilitate the development of scientific software that is efficient, reliable, and broadly available.
-Specifically, we intend for both the library and the software built with it to meet the following goals:
+Empirical is a C++ library designed to promote open science and facilitate the development of scientific software that is efficient, reliable, and universal distributability to researchers and non-experts alike.
```

Comment:
`universal distributability to researchers and non-experts alike` Something's off here, making it a little confusing. Maybe `universal distributability` => `easily distributed`?